nevergrad/functions/pbo/core.py

```diff
@@ -1,26 +1,27 @@
-from IOHexperimenter import IOH_function, W_model_function
 import numpy as np
+from IOHexperimenter import IOH_function, W_model_function
 import nevergrad as ng
-from nevergrad.common.typing import ArrayLike
 from .. import base
 
+
 class PBOFunction(base.ExperimentFunction):
     """
     Pseudo-boolean functions taken from the IOHexperimenter project, adapted for minimization
-    
+
     Parameters
     ----------
     fid: int
-        function number (for a list of functions, please see the documentation)
+        function number (for a list of functions, please see the documentation https://arxiv.org/abs/1810.05281)
```

Comment:
is that the right paper?

Reply:
It is probably better to use this one: https://www.sciencedirect.com/science/article/abs/pii/S1568494619308099